hydra-head.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'hydra-access-controls', version
   s.add_dependency 'hydra-core', version
-  s.add_dependency 'rails', '>= 5.2', '< 6.1'
+  s.add_dependency 'rails', '>= 5.2.5', '< 6.1'
```

Comment:
As discussed on the tech call, let's revert this change and instead add a note to the README or the wiki (possibly https://github.com/samvera/hydra-head/wiki/For-Contributors)